### ISSUE TITLE
zwave: restart daemon on a crash

### ISSUE THREAD

**Issue by uSpike** [home-assistant/hassio-addons#1405]
There seems to be some instability in the qt-ozwdaemon application still.  When it crashes:
```
[20200613 14:51:23.317 CDT] [ozw.daemon] [warning]: =============================
[20200613 14:51:23.317 CDT] [ozw.daemon] [warning]: CRASH!!! - Dumping Backtrace:
[20200613 14:51:23.317 CDT] [ozw.daemon] [warning]: =============================
[20200613 14:51:23.317 CDT] [ozw.daemon] [warning]: #1  0x00007f24cc4b527d sp=0x00007f24c94b4e80 sigwaitinfo + 0x8
[20200613 14:51:23.317 CDT] [default] [warning]: Exiting....
1592077883: Socket error on client qt-openzwave-1, disconnecting.
In exit
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
s6-svwait: fatal: supervisor died
1592077883: mosquitto version 1.6.8 terminating
1592077883: Saving in-memory database to /data/mosquitto.db.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

Is it possible or a good idea to have the service restart if it fails/crashes?

**Comment by frenck** [comment]
This is the ozw daemon crashing, as add-ons wrap existing application, please report your issue with the upstream project:

<https://github.com/OpenZWave/qt-openzwave>

Please make sure to provide larger/full logs, and information about your system (like architectures, boards used).

Thanks 👍

**Comment by uSpike** [comment]
My intent with opening this issue was not to get assistance with the `qt-openzwave` crash.  My concern is that `zwave` is a particularly important component of many home automation setups, and a crashing daemon results in those automation not working.

AFAIK there's not an easy way to get a notification of such a crash.  Is it possible or a good idea to get a notification that the addon has stopped, or automatically restart the addon?  Or, would you rather we just ignore this issue and assume the application should not crash?

**Comment by frenck** [comment]
> My intent with opening this issue was not to get assistance with the qt-openzwave crash. My concern is that zwave is a particularly important component of many home automation setups, and a crashing daemon results in those automation not working.

I agree, please report your issue upstream so it can be fixed.

Your other question is a support question, please use or Community forums or Discord chat for those kind of things. 

Thanks 👍

**Comment by uSpike** [comment]
Fixed in home-assistant/supervisor#1970
